chain-signatures/node/src/web/mod.rs

```diff
@@ -47,8 +47,13 @@ pub async fn run(
         )
         .route("/msg", post(msg))
         .route("/state", get(state))
-        .route("/metrics", get(metrics))
-        .layer(Extension(Arc::new(axum_state)));
+        .route("/metrics", get(metrics));
+
+    if cfg!(feature = "bench") {
```

Comment:
Since we are exposing a new bench URL, should we simplify this and remove the feature flag usage?